### ISSUE TITLE
incompatibility of the tensorflow version with the numpy version

### ISSUE THREAD

**Issue by rania-o** [comprna/CHEUI#11]
Hi developer,

I tried to install CHEUI and its dependencies, but when I installed tensorflow-gpu==2.4.1, numpy is downgraded to version1.19.5. 
Is it normal that the versions of tensorflow-gpu and numpy that you're recommending are not compatible ?
I haven't tested CHEUI yet, I hope it won't cause any errors when running the different scripts.

Thank you,
Rania

**Comment by EduEyras** [comment]
Hi,

it should be fine if the TensorFlow is installed properly.

Best

E.

**Comment by rania-o** [comment]
Hi @EduEyras 

Thanks for your fast reply.
I have another question, I would like to test CHEUI and other tools on the IVT sequences you used in the paper.
I was wondering where you got the modified fast5 m6A from Jenjaroenpun, P. et al. because I did not find a m6A fast5 file in this link :
https://trace.ncbi.nlm.nih.gov/Traces/index.html?view=study&acc=SRP166020

Rania

**Comment by Akanksha2511** [comment]
Hi Rania, 

Thanks for testing CHEUI. Yes, if the fast5 are not available to download from the link you will have to contact the authors. 

Thanks, 
Akanksha

**Comment by rania-o** [comment]
Hi @Akanksha2511 

Thank you for your reply.
I already did but they're not reactive :/ 

Rania

**Comment by rania-o** [comment]
thank you @Akanksha2511 

I have one last question, I saw that the reads you used for training models 1 and 2 from _Liu et al_ were obtained with the RNA001 kit. Wouldn't this be problematic since now all the dRNA sequences (as well as the data you used for your tests) are obtained with the RNA002 kit ?

Rania

**Comment by EduEyras** [comment]
Hi Rania,

That's a good question. We looked into the potential effects of the kit on
the signals and we could not see differences that could be due to the kit,
or if there are, these are difficult to separate from the experimental
variability from the nanopore sequencing.

At the rate of variability and accuracy that we have, we don't see any
effects.

cheers

Eduardo

**Comment by rania-o** [comment]
Hi @EduEyras 

thank you for your answer. I was curious because I saw that for Nanopolish polyA they had indeed detected a difference between the two kits [https://github.com/jts/nanopolish/issues/545], but maybe this difference only concerns the polyA tail.

Rania

**Comment by pre-mRNA** [comment]
Numpy recommendation downgraded to 1.19.5. Thanks Rania!
